src/collections/_documentation/workflow/releases/release-automation/circleci/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: CircleCI
+sidebar_order: 1
+
+---
+
+This guide walks you through the process of automating Sentry release management and deploy notifications in CircleCI. After deploying in CircleCI, you'll be able to identify suspect commits that are likely the culprit for new errors. You'll also be able to apply source maps to see the original code in Sentry.
+
+Before starting, confirm that your Sentry project is properly set up to track commit metadata. The easiest method to do so is to [install a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration). Once that's installed, and you've added your repository, come back to this guide. If you've already installed a repository integration, you're ready to go.
+
+## Create a Sentry Internal Integration
+
+For CircleCI to communicate securely with Sentry, you'll need to create a new internal integration. In Sentry, navigate to **Settings > Developer Settings > New Internal Integration**.
+
+Give your new integration a name (for example, “CircleCI Deploy Integration”) and specify the necessary permissions. In this case, you need Admin access for “Release” and Read access for “Organization”.
+
+For more details about scopes and API endpoints, see the full documentation on [API Permissions]({%- link _documentation/api/permissions.md -%}).
+
+[{% asset releases/circleci/internal-integration-permissions.png alt="View of internal integration permissions." %}]({% asset releases/jenkins/internal-integration-permissions.png @path %})
+
+Click “Save” at the bottom of the page and grab your token, which you'll need this in the next step.
+
+## Setting Environment Variables in CircleCI
+
+Next, you'll need a few environment variables to configure the Sentry CLI:
+
+- `SENTRY_AUTH_TOKEN` - Your internal integration token.
+- `SENTRY_ORG` - Your Sentry organization slug.
+- `SENTRY_PROJECT` - Your Sentry project slug.
+
+To access you internal integration token securely, store it as an [environment variable on your project](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project):
```

Comment:
Is it possible to use the [{% ... %}] link structure here? It's a better experience for readers in case this content moves.

Reply:
that's circleci docs